auth4genai/how-tos/create-spotify-playlist.mdx

```diff
@@ -47,7 +47,7 @@ export async function POST(req) {
           description: z.string().describe("Optional description of the playlist"),
         }),
         execute: async ({ description }) => {
-          const { token: accessToken } = await auth0.getAccessTokenForConnection({ connection: "spotify-custom" });
+          const { token: accessToken } = await auth0.getAccessTokenFromTokenVault({ connection: "spotify-custom" });
```

Comment:
This is a bit tricky to notice... but the `auth0.getAccessTokenForConnection()` method here actually comes from the `@auth0/nextjs-auth0/server` package, not our AI Framework sdks... So the renaming of the method is not applicable here. There's a few other references that were renamed unfortunately in this PR and we should review to only update those methods that reference the AI Framework SDKs

Reply:
updated: 6c1b1844e11d63381d47a6bf8bac03f03bdc919a

it seems the `auth0.getAccessTokenForConnection({ connection: 'google-oauth2' });` (connection param) is the giveaway here..